src/com/yetanalytics/lrs_admin_ui/input.cljs

```diff
@@ -0,0 +1,33 @@
+(ns com.yetanalytics.lrs-admin-ui.input
+  (:require [cljs.spec.alpha :as s :include-macros true]
+            [com.yetanalytics.lrs-admin-ui.functions.password :as pass]
+            [clojure.set :refer [intersection]]))
```

Comment:
Guess we don't need `intersection` anymore.

Reply:
valid

Reply:
fixed